src/page/index/login.tsx

```diff
@@ -0,0 +1,103 @@
+import { zodResolver } from "@hookform/resolvers/zod";
+import {
+  Container,
+  TextField,
+  Button,
+  Box,
+  Typography,
+  CircularProgress,
+} from "@mui/material";
+import { useMutation } from "@tanstack/react-query";
+import { useForm } from "react-hook-form";
+import { useNavigate } from "react-router-dom";
+
+import { authApi } from "@/api/auth/request";
+import { SignInRequestSchema, SignInResponse, type SignInRequest } from "@/type/auth";
+import { setCookie } from "typescript-cookie";
+
+const defaultValues: SignInRequest = {
+  email: "",
+  password: "",
+};
+
+const Login = () => {
+  const navigate = useNavigate();
+
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm<SignInRequest>({
+    resolver: zodResolver(SignInRequestSchema),
+    defaultValues,
+  });
+  
+  const requestSignIn = useMutation({
+    mutationFn: authApi.signIn,
+  })
+
+  const onSubmit = async (data: SignInRequest) => {
+    requestSignIn.mutate(data, {
+      onSuccess: (data: SignInResponse) => {
+        setCookie("Authorization", data.accessToken);
+        navigate(`/academy`);
+      },
+      onError: () => {
+        alert("아이디 비밀번호를 확인해주세요");
```

Comment:
alert 대신 toast 사용하시면 돼요 (근데 혹시 토스트가 안 떠서 따로 넣으신 건지..?!)

```suggestion
        toast.error("아이디 비밀번호를 확인해주세요");
```